olm/quickstarts/rhosak-openshift-quarkus-bind-quickstart.yaml

```diff
@@ -3,223 +3,230 @@ kind: ConsoleQuickStart
-            1. Click on **Display Token**.
-            1. Copy the `oc login` command.
-            1. Go to your terminal, and execute the command you just copied.
+        1. On the command line, ensure that the current OpenShift project is the one in which you deployed the example Quarkus application, as shown in the following example:
 
-        With your `oc` client connected, point it to the correct project:
-        1. In your terminal, point your **oc** client to the correct OpenShift namespace/project (replace *{username}* with your username in the Developer Sandbox OpenShift environment): `oc project {username}-dev`. This should be the project in which you've deployed your Quarkus application and that contains the `KafkaConnection`.
-        1. Execute the command `oc get deployment`. This should returen the **Deployment** of you Quarkus application, and the **Deployment** of your tools container (if you're using that container).
+           ```
+           $ oc project my-project
+           ```
 
-      review:
-        instructions: |-
-          #### Verify that you've successfully connected your `oc` client to your OpenShift instance:
-          Did you see the **Deployment** of your Quarkus application as output of the `oc get deployment` command?
-        failedTaskHelp: This task isn’t verified yet. Try the task again.
-      summary:
-        success: >-
-          Great work! You have access to the tooling required for the remainder of this quickstart.
-        failed: Try the steps again.
-    - title: Bind your Quarkus application to your Streams for Apache Kafka instance.
-      description: |-
-        With your Quarkus application running, and your Streams for Apache Kafka instance connected to your namespace, you can now bind your application to your Kafka instance.
-        This is done using the Service Binding Operator, which will inject the configuration values required to connect to your Kafka instance into your Quarkus application.
-        The Quarkus application has been configured to use the `quarkus-kubernetes-service-binding` [extension](https://github.com/DuncanDoyle/quarkus-kafka-quickstart-summit2021/blob/main/pom.xml#L82-L85)
-        enabling auto-discovery of the binding files injected into the Quarkus application pod.
+        1. Use the RHOAS CLI to bind your Kafka instance to your OpenShift project.
+
+           ```
+           $ rhoas cluster bind
+           ```
+
+           You're prompted to specify the Kafka instance in Streams for Apache Kafka that you want to connect to OpenShift.
 
-        We will create the binding using the `rhoas` command line tool. In future versions of OpenShift, you will also be able to do this directly from the OpenShift Developer Console.
+        1. Type the name of the Kafka instance that you want to connect to OpenShift. Press **Enter**.
```

Comment:
Do you need to type the name, I thought you could simply select it from the list ...?

Reply:
@DuncanDoyle The reason for this was that I found this selection mechanism on the command line a little clunky and unintuitive. So I thought it was cleaner to instruct users to just filter to a single instance by typing the name. 

But I ran through the workflow again a short time ago and it's not so bad. I'll make an update to describe how to highlight a particular instance and hit Enter.